__tests__/app.test.js

```diff
@@ -45,4 +45,39 @@ describe('/api', () => {
             expect(endpoints).toEqual(allEndpoints)
         })
     })
+})
+
+describe('/api/articles/:article_id', () => {
+    test('GET: 200 send a single article object with correct properties', () => {
+        return request(app).get('/api/articles/1')
+        .expect(200)
+        .then(({ body }) => {
+            const { article } = body
+            expect(article.length).toBe(1)
+            article.forEach((article) => {
+                expect(article).toHaveProperty('article_id')
+                expect(article).toHaveProperty('title')
+                expect(article).toHaveProperty('topic')
+                expect(article).toHaveProperty('author')
+                expect(article).toHaveProperty('body')
+                expect(article).toHaveProperty('created_at')
+                expect(article).toHaveProperty('votes')
+                expect(article).toHaveProperty('article_img_url')
+            })
```

Comment:
shouldn't need to forEach here if you return rows[0] in the model, it should just be one article rather than an array of them. 

I would be more specific with the testing, at a minimum I would check that article.article_id is 1 as that proves you have the article you have asked for. We should be able to assert exactly what the article looks like